uint8 packet_type
uint8 packet_size
float32[] packet_data
